roadmap/4_governance_nns/upcoming/2_sns_treasury_safeguards.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Safe guards for the access to the SNS treasury (short-term)
+links:
+  Forum Post: https://forum.dfinity.org/t/safe-guards-for-the-access-to-the-sns-treasury/19669
+  Proposal:
+eta: Q3 23
+is_community: false
+---
+Via the proposal action “Transfer SNS Treasury funds” an SNS can vote on transferring funds from its treasury, for example to pay developers for work on code extensions. Currently, there are no special restrictions in place on this type of proposals. There is the risk that a malicious treasury transfer is approved because:
```

Comment:
```suggestion
Via the proposal action “Transfer SNS Treasury funds” an SNS can vote on transferring funds from its treasury, for example to pay developers for work on code extensions. Currently, this type of proposals follows the same rules as all proposals which can be adopted with only 3% voter participation. There is the risk that a malicious treasury transfer is approved because:
```